local_net/README.md

```diff
@@ -0,0 +1,53 @@
+# Local Net Quick Start
+
+Before running any of the scripts below, ensure you have built Noble using `make build`.
+
+- [Single Validator Network](#single-validator-network)
+- [Multi Validator Network](#multi-validator-network)
+- [In-Place Testnet Fork](#in-place-testnet-fork)
```

Comment:
I think `In-Place Network Fork` sounds better with the other two above, plus we support both testnet and mainnet in the script 😄

Reply:
Agree!
[`ad7ebdc` (#490)](https://github.com/noble-assets/noble/pull/490/commits/ad7ebdcf82b8451c5a98097d995bb30462300300)